src/evaluator/evaluator/eufs_adapter.py

```diff
@@ -59,6 +66,20 @@ def __init__(self, node: rclpy.node.Node):
         )
 
         self._time_sync_.registerCallback(self.state_estimation_callback)
+    
+    def simulated_planning_callback(
```

Comment:
Isn't simulated planning from mocker node? What is this exactly?

Reply:
I used that nomenclature to be consistent with the remaining nodes. The idea was just to state that at the beginning of control computation. I'll fix that